vite-project/src/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { useState } from 'react';
+import TodosLogic from './components/TodosLogic';
+function App() {
+  return (
+    <div>
+      <h1>Todo List</h1>
+      <TodosLogic />
```

Comment:
You can do away with this codebase as it it causes the double rendering of the input section and todo list as below
![Screenshot from 2023-07-20 13-11-33](https://github.com/leloufadel/Todolist-React/assets/100193711/d2f95cfb-a509-4631-b85c-117266f6a8ed)